packages/schemas/tables/_after_all.sql

```diff
@@ -1,5 +1,11 @@
 /* This SQL will run after all other queries. */
 
+---- Create trigger to set tenant name ----
+create trigger set_tenant_name_trigger
```

Comment:
So, the tenant won't have a custom name on creating? It seems that this will override the value.

Reply:
Good call, intended to add a trigger condition when adding PATCH API. Can also do it here.

Reply:
Updated and tested with DB.